locales/ja-JP.yml

```
@@ -83,6 +83,9 @@ files: "ファイル"
 download: "ダウンロード"
 driveFileDeleteConfirm: "ファイル「{name}」を削除しますか？このファイルを使用した一部のコンテンツも削除されます。"
 unfollowConfirm: "{name}のフォローを解除しますか？"
+cancelFollowRequestConfirm: "{name}へのフォロー申請をキャンセルしますか？"
+rejectFollowRequestConfirm: "{name}からのフォロー申請を拒否しますか？"
+rejectFollowRequestConfirmDescription: "拒否したことは相手には通知されません。"
```

フォロー申請の「拒否」は(他サービスにあるような)「無視」とは違って相手から見るとフォロー申請が取り消されているように見えたはずだから「通知されない」というと誤解を招くかも

---

プッシュ通知はされない（けど手動で確認したらわかる）からなんて言うのがいいか

---

相手に通知はされませんが、申請中の状態は解除されます。

とか

---

なんていうのが適切なんだろう

---

特筆しなくても良いような気もしますね…(通知するサーバーは無かったような気もしますが、相手方の実装による部分は一応ありますし)

書くとすると、
```suggestion
rejectFollowRequestConfirmDescription: "拒否するとフォロー申請される前の状態に戻ります。"
```
みたいな形とかでしょうか…

---

> 相手方の実装による部分

確かに
特筆しないでいいか

---

けした